erpnext/accounts/doctype/bulk_statement_of_accounts/bulk_statement_of_accounts.json

```diff
@@ -0,0 +1,284 @@
+{
+ "actions": [],
+ "allow_workflow": 1,
+ "autoname": "Prompt",
+ "creation": "2020-05-22 16:46:18.712954",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "section_break_11",
+  "from_date",
+  "company",
+  "account",
+  "group_by",
+  "cost_center",
+  "column_break_14",
+  "to_date",
+  "finance_book",
+  "currency",
+  "project",
+  "preferences",
+  "include_ageing",
+  "ageing_based_on",
+  "section_break_14",
+  "orientation",
+  "section_break_1",
+  "enable_auto_email",
+  "section_break_18",
+  "frequency",
+  "filter_duration",
+  "column_break_21",
+  "start_date",
+  "section_break_3",
+  "cc_to",
+  "customer_collection",
+  "column_break_6",
+  "primary_mandatory",
+  "section_break_27",
+  "collection_name",
+  "column_break_29",
+  "fetch_customers",
+  "column_break_17",
+  "customer_list"
```

Comment:
We generally use the child table fieldname as the plural form of most important object the child table holds like items, taxes etc. In this case, it should be "customers".